modules/UI/videolayout/FilmStrip.js

```diff
@@ -12,6 +12,46 @@ const FilmStrip = {
     init (eventEmitter) {
         this.filmStrip = $('#remoteVideos');
         this.eventEmitter = eventEmitter;
+        this.filmStripIsVisible = true;
+        this.renderFilmstripToolbar();
+        this.activateHideButton();
+    },
+
+    /**
+     * Attach 'click' listener to "hide filmstrip" button
+     */
+    activateHideButton () {
+        $('#videospace').on('click', '#hideVideoToolbar', () => {
+            var icon = document.querySelector('#hideVideoToolbar i');
+
+            this.filmStripIsVisible = !this.filmStripIsVisible;
+            this.toggleFilmStrip(this.filmStripIsVisible);
+
+            icon.classList.remove(
+                this.filmStripIsVisible ? 'icon-menu-up' : 'icon-menu-down');
```

Comment:
icon-menu-up and icon-menu-down can be variables, so that we make sure we don't do a type mistake on the next line.